backend/src/main/java/io/diveni/backend/model/PersonalAccessToken.java

```diff
@@ -0,0 +1,15 @@
+package io.diveni.backend.model;
+
+import lombok.AllArgsConstructor;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+@Getter
+@EqualsAndHashCode
+@AllArgsConstructor
+@NoArgsConstructor
+public class PersonalAccessToken {
+
+  String code;
```

Comment:
Do we need to create PersonalAccessToken objects without parameters, or do we want to modify the PersonalAccessToken code after object creation? If not, we probl. could also set the model to be immutable, for example:

```
@Getter
@EqualsAndHashCode
@RequiredArgsConstructor
public class PersonalAccessToken {
    private final String code;
}
```